### DISCUSSION TITLE
Transit Duration and Azimuth

### DISCUSSION THREAD

**Discussion by abdeldjalil-fellah** [Stellarium/stellarium#382] [Q&A] [answered]
Latitude: 36.0
Date: 25/09/2018
Sun Apparent Diameter: **0.53163**

When the West Edge of the Sun Disk crosses the Transit Line (Az = 180.0):
the expected Sun Azimuth is: 180.0 - **0.53163**/2 = 179.734185 but Stellarium shows : 179.556

When the East Edge of the Sun Disk crosses the Transit Line (Az = 180.0):
the expected Sun Azimuth is: 180.0 + **0.53163**/2 = 180.265815 but Stellarium shows : 180.4472

![sun](https://user-images.githubusercontent.com/6022170/46002712-afb85f00-c0a6-11e8-93b0-f9e6215e54b4.jpg)

**Comment by gzotti** [comment, marked as answer]
You forgot that azimuth lines converge towards the zenith.

Latitude 36, noon altitude 53°. 
180.0+0.53163/2/cos(53°)=180.44...
